### PR TITLE
Fix travis 1212

### PR DESCRIPTION
**Pull request by csantanapr**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
     - secure: "cZijVhCVtuBSAH/EpjIxfXSjZGjRULM5UYyvKkDZx/FqMd47K0+xzDWIfizowvFa4bGjU2JgjklmlERdF/P3Xp34UDciLvt7q274G8Rqnlq52KMM56iL/LbkeISeaRgbN7lWq+XZnkRDojAIrM67HUn9lsYs4SXAY8HCBXPnzhLN8dRE7BbEGKjQJceUQMxKNfiCVJ7r29qFXbdtjMQ8ea6lHJkczUaLKEFWYGpqIGXJMYAhDGd54OuwqrjOQR1AtNGjyEy1yJRoexKemfrvYl2kUuH6qtJwQv4dQMpSOkVtHOISoiIyHjsxYnCGgKnrLRz+R2F9I1PFxZaVlJEMrG3bc7CDZbXowcnmNEy5FxdAoxgxaTWZwO8mVRnaY/u5hEOkz23Hm/Kmkq5d17XThSrwQh9GG3DZMGabINwo05yAKjSmFV1wbWZmIYzU2ls1H2P+T833TkYqV5PQfOXYFfpV7jrBT0fydgIMIEmPp20Bejf+2KvDcn8ITwR8u25g7TIkmp0CUgv5blWowtGLBH08Ps/yM4YKXX20nNuuAWVYN3OzFAHmtHHAAGaEj0z5xtpKjwSKInFQtuAa3uTG5qMb2/xBxRKhsJmXYVIYRDsGg1Kmk/G97vJSM0WZykUt3EyvfQe4xq9UtHSO5jtnVl4XxXp6gQOTWSuZyUVpIHI="
-services:
-  - docker
+
```
Comment (dubeejw) [inline .travis.yml]: Shouldn't need the extra new line here.

File: .travis.yml [hunk 2]
```diff
@@ -1,16 +1,16 @@
 # A Travis CI configuration file.
 
 sudo: required
-
+group: deprecated-2017Q3
 language: scala
-scala:
-   - 2.11.6
+services:
+- docker
```
Comment (dubeejw) [inline .travis.yml]: Tab here?

Comment (csantanapr) [reply .travis.yml]: no tab need it

File: .travis.yml (hunk 1 above)
Comment (csantanapr) [reply .travis.yml]: submitted commit, to remove the extra line

**Pull request merged**